Reference/Searching/Examine/quick-start.md

```diff
@@ -55,74 +56,54 @@ In MVC we have a method called `TypedSearch` on the `UmbracoHelper` which will r
                 </li>
             }
         </ul>
-    }   
+    }
 
-### Razor
+### ExamineManager
 
-In razor macros there's a `Search` method on the `DynamicNode` model which will return a `DynamicNodeList`:
+To complete more complex searches, we can use the ExamineManager and our custom search provider to expose further functionality, such as being able to handle mispellings, for example.
 
-    @if (!string.IsNullOrEmpty(Request.QueryString["query"]))    
+    var query = Request.QueryString["query"];
+
+    var searcher = Examine.ExamineManager.Instance.SearchProviderCollection["ExternalSearcher"];
+    
+    // the boolean parameter is whether to use wildcards when searching.
+    var searchResults = Searcher.Search(query, true).OrderByDescending(x => x.Score);
+    if(searchResults.Any())
     {
         <ul>
-            @foreach (var result in Model.Search(Request.QueryString["query"]))
+            @foreach (var result in searchResults)
             {
                 <li>
                     <a href="@result.Url">@result.Name</a>
                 </li>
             }
         </ul>
-    } 
+    }
+
+### Fluent API
 
-### Web forms
+Examine offers a fluent search API which aims to make constructing complex searches simple. The underlying API is determined by the provider implementation, with Examine just exposing the appropriate methods. With the fluent API, we can introduce fuzzy-text search and boosting, which allows us to favour a search result if our search term is found in particular fields, including custom fields added to document types. 
 
-We can use almost the same format as the above in our master pages if you are using webforms. However we're using the `ExamineManager` directly to do our search which returns an `ISearchResult`:
+    var query = Request.QueryString["query"];
+    var searcher = Examine.ExamineManager.Instance.SearchProviderCollection["ExternalSearcher"];
 
-    <%if (!string.IsNullOrEmpty(Request.QueryString["query"])) { %>
+    var searchCriteria = searcher.CreateSearchCriteria(Examine.SearchCriteria.BooleanOperation.Or);
+    var searchQuery = searchCriteria.Field("nodeName", query.Boost(5)).Or().Field("nodeName", query.Fuzzy());
+    var searchResults = searcher.Search(searchQuery.Compile()).OrderByDescending(x => x.Score);
```

Comment:
Same remark about the OrderByDescending().  

Oh, and they are ordered by score because that is how lucene works.  Right?  Correct me if I am wrong.

Reply:
True - switched them to use the `createDate`, just an example of ordering the results.